src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -48,8 +49,22 @@ internal sealed partial class SmartRenameViewModel : INotifyPropertyChanged, IDi
     public string StatusMessage => _smartRenameSession.StatusMessage;
 
     public bool StatusMessageVisibility => _smartRenameSession.StatusMessageVisibility;
-    public bool IsUsingResultPanel { get; set; }
-    public bool IsUsingDropdown { get; set; }
+
+    public bool SupportsAutomaticSuggestions { get; }
+
+    public bool IsAutomaticSuggestionsEnabled { get; private set; }
+
+    public void ToggleAutomaticSuggestions()
+    {
+        if (!SupportsAutomaticSuggestions)
```

Comment:
if Toggling already checks if we support this, then why do the callers *also* check this?